src/plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -6,5 +6,13 @@ export const mappings = {
         index: 'not_analyzed'
       }
     }
+  },
+  server: {
+    properties: {
+      uuid: {
+        type: 'string',
```

Comment:
I think this mapping should be updated so it's not using the deprecated string mappings.  See https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking_50_mapping_changes.html#_literal_string_literal_fields_replaced_by_literal_text_literal_literal_keyword_literal_fields